LICENSE

```diff
@@ -211,3 +211,96 @@ Home page: https://github.com/gradle/gradle
 License: https://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
+
+This product includes code from Apache Iceberg.
+
+* spec/rest-catalog-open-api.yaml
+* integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisRestCatalogIntegrationTest.java
+
+Copyright: Copyright 2017-2025 The Apache Software Foundation
+Home page: https://iceberg.apache.org
+License: https://www.apache.org/licenses/LICENSE-2.0
+
+--------------------------------------------------------------------------------
+
+This product includes code from OpenAPITool openapi-generator
+
+* server-templates/formParams.mustache
+* server-templates/apiService.mustache
+* server-templates/bodyParams.mustache
+* server-templates/pojo.mustache
+* server-templates/headerParams.mustache
+* server-templates/apiServiceImpl.mustache
+* server-templates/queryParams.mustache
+* server-templates/api.mustache
+
+Copyright: Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech) Copyright 2018 SmartBear Software
```

Comment:
Newline for the second copyright?

Reply:
Yup, it's would be cleaner 😄 Doing the change.